.github/scripts/trymerge.py

```diff
@@ -1223,26 +1281,27 @@ def merge(pr_num: int, repo: GitRepo,
             raise RuntimeError("New commits were pushed while merging. Please rerun the merge command.")
         try:
             find_matching_merge_rule(pr, repo)
```

Comment:
I think we can remove some of the instances of this since we don't need to keep checking the find_matching_merge_rule

Reply:
Yup, I want to put that in a separate PR though